mbuild/formats/lammpsdata.py

```diff
@@ -169,6 +180,9 @@ def write_lammpsdata(structure, filename, atom_style='full',
     if use_rb_torsions and use_dihedrals:
         raise ValueError("Multiple dihedral styles detected, check your "
                          "Forcefield XML and structure")
+    if mins == None:
+        print("Box information was not passed to the save function,\n"
```

Comment:
May be use a `warn` here instead of print?

Reply:
- I don't think there would be a case where mins/maxs and length don't match up, since they all (both mins/maxs and structure.box) are parsed from the mbuild box class.  But it would be good to have a check there anyway. 

- I just did a print statement to be consistent with the way the if detect_forcefield_style stuff was written (e.g., see line 160).  Do we want to convert all those print statements to warnings? Are these really warnings or more just notes? Is there a distinction we are making between these?

Reply:
Thinking about this some more....structure.box and and the mins and maxs don't actually have to be the same.  They should be the same if you call "save" (since they both are extracted from the same mbuild box class). However, if I were to use the apply command (e.g., to a single molecule), the box info in the parmed structure is just the bounds of the molecule, not necessarily the bounds of the system box.  For example, in the code below (for the unit test I'm adding in), the mins and maxs would absolutely not define the same box size as structure.box, and that would be by design. 


`
from foyer import Forcefield
from mbuild.lib.molecules import Ethane
from mbuild.formats.lammpsdata import write_lammpsdata

OPLSAA = Forcefield(name='oplsaa')
structure = OPLSAA.apply(Ethane())
box = mb.Box(mins=np.array([-1.0, -2.0, -3.0]), maxs=np.array([3.0, 2.0, 1.0]))
write_lammpsdata(filename='box.lammps', structure=structure,
                 unit_style='real', mins=[m for m in box.mins], maxs=[m for m in box.maxs])`